spec.md

```diff
@@ -546,6 +548,10 @@ The tags MUST be in lexical order.
 
 When using the `last` query parameter, the `n` parameter is OPTIONAL.
 
+The registry SHOULD support at least `n=1000` tags in a single response.
```

Comment:
```suggestion
Registries MAY restrict the maximum number of tags to be returned. For example, a limit of `n=1000` tags in a single response.
```

Reply:
I removed the recommended limit from the PR.